tests/layer_tests/ovc_python_api_tests/test_tf.py

```diff
@@ -1045,4 +1049,110 @@ def test_mo_convert_tf_model(self, params, ie_device, precision, ir_version,
         ref_model = params['ref_model']
 
         test_params.update({'input_model': fw_model})
-        self._test_by_ref_graph(temp_dir, test_params, ref_model, compare_tensor_names=False)
\ No newline at end of file
+        self._test_by_ref_graph(temp_dir, test_params, ref_model, compare_tensor_names=False)
+
+
+class TestOutputTensorName(unittest.TestCase):
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    def test_tf1_from_file_single_tensor_name(self):
+
+        Path(constants.out_path).mkdir(parents=True, exist_ok=True)
+        tmp_dir = tempfile.TemporaryDirectory(dir=constants.out_path).name
+
+        from openvino import convert_model
+
+        model, _, _ = create_tf_graph_def(None)
+        path = save_to_pb(model, tmp_dir)
+
+        ov_model = convert_model(path)
+        out_tensors = ov_model.outputs[0].get_names()
+
+        assert len(out_tensors) == 1
+        assert list(out_tensors)[0].endswith(":0")
```

Comment:
can we check that tensor name exactly equal to expected value instread of just containing `:0`?

Reply:
Done.